.travis.yml

```diff
@@ -76,6 +77,12 @@ install:
         ;;
     esac
 
+before_script:
+  # fail the build if there are Python syntax errors or undefined names
+  - flake8 . --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+  # exit-zero treats all errors as warnings.  The GitHub editor is 127 chars wide
+  - flake8 . --count --exit-zero --max-complexity=10 --max-line-length=127 --statistics
```

Comment:
This run seems to generate 95% "E111 indentation is not a multiple of four" warnings which drown out the rest.

TensorBoard uses a 2-space indent for code blocks (vs 4 spaces for continuation lines) since that's what internal google style is and what TensorFlow itself uses, though I guess the official external google python style guide specifies 4 spaces.  But I think converting the whole codebase from 2 to 4 space block indentation is pretty low on our priorities especially if TensorFlow is sticking with 2 spaces.

So let's exclude E111 for now, since I don't think reporting it a hundred lines per file is useful.  Once that isn't being reported, there may be a few others that make sense to ignore by default.